internal/resolvers/imgur/load_test.go

```diff
@@ -0,0 +1,365 @@
+package imgur
+
+import (
+	"net/url"
+	"testing"
+	"time"
+
+	"github.com/Chatterino/api/internal/mocks"
+	qt "github.com/frankban/quicktest"
+	"github.com/golang/mock/gomock"
+	"github.com/koffeinsource/go-imgur"
+)
+
+func testLoadAndUnescape(c *qt.C, urlString string) (cleanTooltip string) {
+	iret, _, err := load(urlString, nil)
+
+	c.Assert(err, qt.IsNil)
+	c.Assert(iret, qt.Not(qt.IsNil))
+
+	response := iret.(response)
+
+	c.Assert(response, qt.Not(qt.IsNil))
+	c.Assert(response.err, qt.IsNil)
+
+	c.Assert(response.resolverResponse, qt.Not(qt.IsNil))
+
+	cleanTooltip, unescapeErr := url.PathUnescape(response.resolverResponse.Tooltip)
+	c.Assert(unescapeErr, qt.IsNil)
+
+	return cleanTooltip
+}
+
+func TestLoad(t *testing.T) {
+	datetime := time.Date(2019, time.November, 10, 23, 0, 0, 0, time.UTC).Unix()
+	c := qt.New(t)
+	mockCtrl := gomock.NewController(c)
+	m := mocks.NewMockImgurClient(mockCtrl)
+	apiClient = m
+
+	c.Run("Normal image", func(c *qt.C) {
+		const url = "image"
+		m.
+			EXPECT().
+			GetInfoFromURL(gomock.Eq(url)).
+			Return(&imgur.GenericInfo{
+				Image: &imgur.ImageInfo{
+					Title:       "My Cool Title",
+					Description: "My Cool Description",
+					Datetime:    int(datetime),
+				},
+				Album:  nil,
+				GImage: nil,
+				GAlbum: nil,
+				Limit:  &imgur.RateLimit{},
+			}, 420, nil)
+
+		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> My Cool Title</li><li><b>Description:</b> My Cool Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li></div>`
+
+		cleanTooltip := testLoadAndUnescape(c, url)
+
+		c.Assert(cleanTooltip, qt.Equals, expectedTooltip)
+	})
+
+	c.Run("NSFW image", func(c *qt.C) {
+		const url = "nsfw_image"
+		m.
+			EXPECT().
+			GetInfoFromURL(gomock.Eq(url)).
+			Return(&imgur.GenericInfo{
+				Image: &imgur.ImageInfo{
+					Title:       "My Cool Title",
+					Description: "My Cool Description",
+					Datetime:    int(datetime),
+					Nsfw:        true,
+				},
+				Album:  nil,
+				GImage: nil,
+				GAlbum: nil,
+				Limit:  &imgur.RateLimit{},
+			}, 420, nil)
+
+		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> My Cool Title</li><li><b>Description:</b> My Cool Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li><li><b><span style="color: red">NSFW</span></b></li></div>`
```

Comment:
Keep consistency with css semicolons
```suggestion
		const expectedTooltip = `<div style="text-align: left;"><li><b>Title:</b> My Cool Title</li><li><b>Description:</b> My Cool Description</li><li><b>Uploaded:</b> 10 Nov 2019 • 23:00 UTC</li><li><b><span style="color: red;">NSFW</span></b></li></div>`
```